### DISCUSSION TITLE
Is there some way to see the value of variables other than in the request log?

### DISCUSSION THREAD

**Discussion by avdi** [Huachao/vscode-restclient#1268] [Q&A] [answered]
Is there some way to view variable values (particularly the ones dynamically assigned from response values) without interpolating them into a request and looking at the logs?

**Comment by avdi** [comment]
Ope, I see that mousing over later references to the variable works. I'm still curious if there's a way to see all variables listed with their current values.

**Comment by wilsong-mb** [comment, marked as answer]
@avdi , in the current version, no, there isn't.
A pretty good feature request
